File: Chapter 1/(e)/e.java
import java.util.Scanner;

class e 
{
	static public void main(String args[])
	{
		Scanner input=new Scanner(System.in);
		float l,b,r;

		System.out.print("Enter the "+"length : ");
		l=input.nextFloat();
		System.out.print("Enter the "+"breadth : ");
		b=input.nextFloat();
		System.out.print("Enter the "+"radius : ");
		r=input.nextFloat();

		System.out.println("Area of "+"rectangle : "+l*b );
		System.out.println("Perimeter of "+"rectangle "+2*(l+b) );
		System.out.println("Area of "+"circle "+3.14*r*r );
		System.out.println("Circumference of "+"circle "+2*3.14*r );

	}
}

File: Chapter 1/(e)/e.c
#include <stdio.h>
#define PI 3.14

int main(int argc, char *args[])
{
	float l, b, r;
	printf("Enter the length ");
	scanf("%f", &l);
	printf("Enter the breadth ");
	scanf("%f", &b);
	printf("Enter the radius ");
	scanf("%f", &r);

	printf("Area of Rectangle : %0.3f\n", l * b);
	printf("Perimeter of Recatangle : %0.3f\n", 2 * (l + b));

	printf("Area of circle : %0.3f\n", PI * r * r);
	printf("Circumference of circle : %0.3f\n", 2 * r * PI);

	return 0;
}